src/components/flat-table/__internal__/use-table-cell.ts

```diff
@@ -19,6 +26,16 @@ export default (id: string) => {
   const isFirstCell = id === firstCellId;
   const isExpandableCell = expandable && isFirstCell && firstColumnExpandable;
 
+  useEffect(() => {
+    if (tabstopTimer.current) {
+      clearTimeout(tabstopTimer.current);
+    }
+
+    tabstopTimer.current = setTimeout(() => {
```

Comment:
**suggestion(not blocking):** Reckon it's worth putting a comment here explaining why the `setTimeout` is needed here? Might be helpful to future maintainers when we're long gone and retired.